### PR TITLE
Add pantry test

### PR DESCRIPTION
**Pull request by kammon10**
#### What's this PR do?
Add pantry test file and update sampleUser pantry

#### Where should the reviewer start?
Pantry-test.js file

#### How should this be manually tested?
none yet

#### Any background context you want to provide?
no

#### What are the relevant tickets?
none

#### Screenshots (if appropriate)
na

#### Questions:
none

### CODE REVIEW

File: src/data/sampleUser.js [hunk 1]
```diff
@@ -19,6 +19,18 @@ const sampleUser = [
         "ingredient": 11215,
         "amount": 5
       },
+      {
+        "ingredient": 19334,
```
Comment (anewb87) [inline src/data/sampleUser.js:23]: Adding ingredients to our sample so that we test happy and sad paths.

File: test/Pantry-test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import User from '../src/classes/User';
+import sampleData from '../src/data/sampleData';
+import sampleIngredients from '../src/data/sampleIngredients';
+import sampleUser from '../src/data/sampleUser';
+
+describe('Pantry', () => {
+  let user;
+
+  beforeEach(() => {
+    user = new User(sampleUser[0], sampleIngredients)
```
Comment (aliroemhildt) [inline test/Pantry-test.js:11]: the pantry for this user will be a pantry instance

**Review by aliroemhildt — APPROVED** [review]
A great start to our Pantry class/tests!

File: test/Pantry-test.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    expect(user.pantry.getMissingIngredients(recipe2).to.deep.equal([
+      {'ingredient': 1009016, 'amount': 1.5},
+      {'ingredient': 20027, 'amount': 1},
+      {'ingredient': 1002046, 'amount': 1},
+      {'ingredient': 20027, 'amount': 1},
+      {'ingredient': 1012046, 'amount': 1},
+      {'ingredient': 19911, 'amount': 0.25},
+      {'ingredient': 16112, 'amount': 1},
+      {'ingredient': 10010062, 'amount': 24},
+      {'ingredient': 1102047, 'amount': 2},
```
Comment (anewb87) [inline test/Pantry-test.js:46]: All other ingredients here and their corresponding amounts were completely missing. This ingredient already exists in the user's pantry BUT the recipe requires 4 and the user's pantry only has 2. We like that this line is required so we can see that our method should work with existing information but update it.

**Review by anewb87 — APPROVED** [review]
Like where this is headed!

**Pull request merged**